compute_tools/src/http/openapi_spec.yaml

```diff
@@ -37,7 +37,7 @@ paths:
               schema:
                 $ref: "#/components/schemas/ComputeMetrics"
 
-  /metrics
+  /metrics:
```

Comment:
oops. We need CI on this file using https://github.com/daveshanley/vacuum. I think we should also explore https://crates.io/crates/oasgen, https://github.com/ProbablyClem/utoipauto, or https://github.com/tamasfe/aide for auto-generating this document.